governance/pm-core-meeting-notes/24Jan24.md

```diff
@@ -0,0 +1,238 @@
+# The Turing Way - Core Project Management Meetings 24 January
+
+## Bi-weekly meetings (with some exceptions)
+
+**Participants** 
+This call is joined by *The Turing Way* Core Staff:
+
+* Kirstie Whitaker
+* Malvika Sharan 
+* Alexandra Araujo Alvarez 
+* Arielle Bennett 
+
+## Agenda
+
+Review together the agenda and agree on priorities and timings.  
+
+:hourglass_flowing_sand: **Timing:** Total 90 min
+
+| Duration | Activity |
+| ---- | -------- |
+| Start | 👋 Welcome |
+| 05 mins | Check in   |
+| 20 mins | Infrastructure WG   
+| 10 mins | TTW Community Call Agenda (15 Feb)
+| 20 min | Collaborations Workshop (TPS & TTW Participation) 
+| 10 min | CZI Localisation Proposal / Jupyterhub updates
+| 20 mins | Other
+5 mins | Wrap up  
+
+## Infrastructure WG
+
+### Buy a URL for The Turing Way 
+
+Documentation here: https://github.com/the-turing-way/the-turing-way/issues/3266
+
+#### Reasons to buy the URL and level of importance:
+- The domain name is given by netlify. If we swap to readthedocs, the URL will find a broken webpage. 
+- If we buy one, we will redirect traffic from one to another url. While we could redirect from netlify to readthedocs, it is much better to type theturingway.org (or the preferable domain). We don't need to rely on netlify or readthedocs.
+- If in the future if there's another platform to deploy, the change will be easier and our URL name will be consistent
+- Proposed planning/design work (developed by Jim):
+    - Domain structure:
+        - root - the-turing-way.org (redirects to start page)
+        - startpage - start.the-turing-way.org (based ond [this](https://the-turing-way.start.page/)?)
+        - book - book.the-turing-way.org
+        - calendar - cal.the-turing-way.org (who and how would this be maintained / automated?) 
+        - etc.
+        - SSL certs (if we need to do that ourselves) - Let's Encrypt
+
+#### Pain points:
+- Maintainance ownership: (as per Jim contribution) "It is worth thinking about bus factor when we plan how to purchase the domain name and manage DNS. We don't want to have to require a particular person to make any changes or fix problems.I like the idea of the Infrastructure WG being responsible."
+- Budget holder / Credit card details: shouldn't this be paid by The Alan Turing? If not, it would be safe to share credentials. Ideas?
+    - theturingway.org is £11.01 first year, £19.97 after (on godaddy)
+    - £5.92 on namecheap
+- Identify and decide the URL name, who and how is this decided?
+    - Examples: theturingway.org / theturingway.io 
+
+### Notes and actions: 
```

Comment:
@AlexandraAAJ - mixing together notes and actions is a little confusing - do you think you could pull out the specific actions please? The notes are great but it makes scanning through to know exactly what the tasks are a little difficult!

Maybe a summary at the top of the notes?

Reply:
Thanks Kirstie, I've added the action on top.